assets/scss/components/vendor/slick/_slick.scss

```diff
@@ -52,14 +52,11 @@
     }
 
     &::before {
-        @include carousel-arrow-color( $slick-arrow-color, 'next')
-    }
+        @include carousel-arrow-color( $slick-arrow-color, "next"); }
```

Comment:
is it expected behaviour when closing curly bracket not on the separate line?

Reply:
@yurytut1993 Autofix issue, fixed